.travis.yml

```diff
@@ -55,3 +63,17 @@ jobs:
       script: bash .travis/deploy.sh common
       on:
         tags: true
+  - stage: publish-beta-docs
```

Comment:
Why a specific beta job and not just have one job for releases and one for nightly?